ionoscloud/resource_user.go

```diff
@@ -377,5 +377,29 @@ func setUserData(d *schema.ResourceData, user *ionoscloud.User) error {
 		}
 	}
 
-	return nil
+	return d.Set("group_ids", getUserGroups(user))
```

Comment:
@digna-ionos let's keep the consistency and let's make the set inside of the function, the function should end with a `return nil` sentence. Is not a big deal, but we have this format for other resources so we should keep it the same way.